invokeai/frontend/web/src/features/settingsAccordions/components/GenerationSettingsAccordion/GenerationSettingsAccordion.tsx

```diff
@@ -46,60 +46,65 @@ const badgesSelector = createMemoizedSelector(
   }
 );
 
-export const GenerationSettingsAccordion = memo(() => {
-  const { t } = useTranslation();
-  const { loraTabBadges, accordionBadges } = useAppSelector(badgesSelector);
-  const { isOpen: isOpenExpander, onToggle: onToggleExpander } =
-    useExpanderToggle({
-      id: 'generation-settings-advanced',
-      defaultIsOpen: false,
-    });
-  const { isOpen: isOpenAccordion, onToggle: onToggleAccordion } =
-    useStandaloneAccordionToggle({
-      id: 'generation-settings',
-      defaultIsOpen: true,
-    });
+interface GenerationSettingsAccordionProps {
+  collapseAdvanced?: boolean;
+}
```

Comment:
If this is a global flag, to be applied to all of the expanders, it probably belongs in `useExpanderToggle`.